components/service/glean/src/main/java/mozilla/components/service/glean/storages/TimingDistributionsStorageEngine.kt

```diff
@@ -206,8 +211,13 @@ data class TimingDistributionData(
                 return null
             }
             val sum = jsonObject.tryGetLong("sum") ?: return null
-            val timeUnit = TimeUnit.values()[jsonObject.tryGetInt("timeUnit")
-                ?: return null]
+            val rawTimeUnit = jsonObject.tryGetString("time_unit") ?: return null
+            var timeUnit: TimeUnit
```

Comment:
As above, we could do without a `var` and assign the return value of the `try` block to `timeUnit`.